blocks/components/editable/index.js

```diff
@@ -347,6 +438,8 @@ export default class Editable extends wp.element.Component {
 				settings={ {
 					forced_root_block: inline ? false : 'p'
 				} }
+				isEmpty={ String( this.isEmpty( value ) ) }
```

Comment:
Even if it's eventually applied as a string, in passing as a prop, `isEmpty` seems to make more sense as a boolean value.